ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -505,6 +505,7 @@ pattern CardanoNodeToClientVersion11 =
       :* EraNodeToClientEnabled ShelleyNodeToClientVersion6
       :* EraNodeToClientEnabled ShelleyNodeToClientVersion6
       :* EraNodeToClientEnabled ShelleyNodeToClientVersion6
+      :* EraNodeToClientEnabled ShelleyNodeToClientVersion7
```

Comment:
This does not typecheck, instead of adding a new component, update the existing versions from `6` to `7`, i.e.
```haskell
-- | The hard fork enabled, and the Shelley, Allegra, Mary, Alonzo, Babbage and
-- Conway eras enabled Using 'ShelleyNodeToClientVersion7' for the
-- Shelley-based eras, which enables new queries.
pattern CardanoNodeToClientVersion11 :: BlockNodeToClientVersion (CardanoBlock c)
pattern CardanoNodeToClientVersion11 =
    HardForkNodeToClientEnabled
      HardForkSpecificNodeToClientVersion2
      (  EraNodeToClientEnabled ByronNodeToClientVersion1
      :* EraNodeToClientEnabled ShelleyNodeToClientVersion7
      :* EraNodeToClientEnabled ShelleyNodeToClientVersion7
      :* EraNodeToClientEnabled ShelleyNodeToClientVersion7
      :* EraNodeToClientEnabled ShelleyNodeToClientVersion7
      :* EraNodeToClientEnabled ShelleyNodeToClientVersion7
      :* EraNodeToClientEnabled ShelleyNodeToClientVersion7
      :* Nil
      )
```

Reply:
In your [latest force push](https://github.com/input-output-hk/ouroboros-network/compare/0112ca3ad92fdbd32cd17effbbe91bfe655660ca..53fb04bb324d17b7e942c3647ce417470e7f5383), you only edited the last Shelley number, and not all of them, as in the [snippet above](https://github.com/input-output-hk/ouroboros-network/pull/4507#discussion_r1171386510). That is not necessarily wrong, but it means that e.g. if there is a node release with Conway support, but the Conway HF has not yet happened, the deposit query will not work as we still have `ShelleyNodeToClientVersion6` for the penultimate era (Babbage).